mne/stats/erp.py

```diff
@@ -32,6 +32,37 @@ def compute_sme(epochs, start=None, stop=None):
     component as the mean within the time window (as opposed to e.g. the peak, which
     would require bootstrapping).
 
+    Examples
+    --------
+    Given an ``epochs`` object, the SME for the entire epoch duration can be computed as
```

Comment:
```suggestion
    Given an :class:`~mne.Epochs` object, the SME for the entire epoch duration can be computed as
```

Reply:
Thanks, I can't commit your suggestion because it exceeds the max line length. I'll commit it separately though!